client/src/main/scala/io/delta/sharing/client/model.scala

```diff
@@ -93,6 +94,13 @@ private[sharing] case class SingleAction(
   }
 }
 
+case class QueryStatus(
+      queryId: String = null,
+      status: String = null
```

Comment:
status should ideally be an enum?
We should perhaps add an error message for server to be able to send errors back to the client.

Reply:
Not needed if  the rpc itself returns error.